build.sh

```diff
@@ -257,6 +230,8 @@ fi
 $QUIET || echo
 $QUIET || echo "Generating spec..."
 $QUIET || echo
+cp -p  $HTML_SOURCE/entities.inc $HTML_CACHE
```

Comment:
`.pre-process-main.pl` includes this file into HTML source. It doesn't mention it by file name unfortunately so it was a little hard to find.